### PR TITLE
ROX-13722: Readd nodeID to NodeInventory proto

### PR DESCRIPTION
**Pull request by vikin91**
## Description

This PR was created as a split from #4430 in order to make the change easier to review.
This PR is meant to replace #3757 - as we will not attach NodeInventory to Node anymore, we need to change the NodeInventory so that it contains the information we will need in Central.

Here, we readd the `nodeID` field to `NodeInventory` as this field will be needed to uniquely identify the Node that the NodeInventory belongs to. The `NodeInventory.NodeID` will be filled by Sensor in another PR. 

Note that by abandoning #3757 and following #4430 we DO NOT:
- attach NodeInventory to Node,
- expose NodeInventory over public APIs,
- save NodeInventory in the Central-DB

anymore.

I also renamed central pipeline for handling Node Inventories from `nodescanv2` to `nodeinventory`.

## Checklist
- [X] Investigated and inspected CI test results

### Not apply
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

## Testing Performed

- CI
- Deploy locally on Colima, enable FF and check log for errors

### CODE REVIEW

**Comment by vikin91** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4483** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4483" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/4483?utm_source=stack-comment).

File: proto/storage/node.proto [hunk 1]
```diff
@@ -84,16 +84,17 @@ message NodeScan {
 }
 
 message NodeInventory {
-    string                         node_name = 1;
-    google.protobuf.Timestamp      scan_time = 2;
+    string                           node_id = 1;
+    string                         node_name = 2;
+    google.protobuf.Timestamp      scan_time = 3;
```
Comment (vikin91) [inline proto/storage/node.proto:89]: No need for backwards compatibility here as this object is fully protected by the feature flag and has not bee used by our customers before

**Review by md2119 — APPROVED** [review]

**Review by jschnath — APPROVED** [review]
LGTM

**Pull request merged**